ietf/static_src/css/iab.scss

```diff
@@ -0,0 +1,200 @@
+// Color swatches:
+$brand-blue: #3e96c8; // (62,150,200)
+$blue: #0b8cc5; // (11, 140, 97)
+$blue-2: #004f73; // (0, 79, 115)
+$navy: #002d3c; // (0, 45, 60)
+$yellow: #ffd13e; // (255, 209, 62)
+
+$grey-1: #f7fafc; // (247, 250, 252)
+$grey-2: #f0f6fa; // (240, 246, 250)
+$grey-3: #e6edf2; // (230, 237, 242)
+$grey-4: #d9e0e5; // (217, 224, 229)
+$grey-5: #cbd2d6; // (230, 210, 214)
+
+.h1-iab {
```

Comment:
Is the `*-iab` classname suffix new, or an existing pattern?

I see you have an element with `btn-secondary-iab` but that element doesn't also have `btn-secondary`. 

But there's also an element with `container` (the bootstrap class) AND `container-iab`.

I'm trying to understand how this naming convention works. How (or if) BS overrides are distinct from all-new code.

Reply:
I've gone through and made that consistent. Now classes that start with `iab-*` are standalones that apply iab specific styling, and `*--iab` are modifier classes intended to be applied elements that also have their non iab equivalent parent class applied.